snapshots/snapshot_generator/src/test/scala/uk/ac/wellcome/platform/snapshot_generator/source/ElasticsearchSourceTest.scala

```diff
@@ -11,6 +11,7 @@ import uk.ac.wellcome.fixtures.TestWith
 import uk.ac.wellcome.elasticsearch.test.fixtures.ElasticsearchFixtures
 import uk.ac.wellcome.models.work.generators.WorkGenerators
 import uk.ac.wellcome.models.work.internal._
+import uk.ac.wellcome.platform.snapshot_generator.akka.source.ElasticsearchWorksSource
```

Comment:
Does this indicate the test needs to move to a different package?

Reply:
common?